DXMainClient/Online/CnCNetManager.cs

```diff
@@ -519,6 +520,11 @@ public void OnGenericServerMessageReceived(string message)
 
         private void DoGenericServerMessageReceived(string message)
         {
+            if (message.Contains("DroneBL"))
```

Comment:
I can't remember what calls `DoGenericServerMessageReceived`. This won't get triggered by any random person sending the message "DroneBL", right?

Reply:
I did just check to make sure, it doesn't no. Its server messages received from GameSurge. 

```
08.09. 21:28:44.599    DoGenericServerMessageReceived **  AUTH *** Looking up your hostname 
08.09. 21:28:44.599    DoGenericServerMessageReceived **  AUTH *** Checking Ident 
08.09. 21:28:44.599    DoGenericServerMessageReceived **  AUTH *** Found your hostname 
08.09. 21:28:52.570    DoGenericServerMessageReceived **  AUTH *** No ident response 
08.09. 21:28:52.639    DoGenericServerMessageReceived ** TAL.DE.EU.GameSurge.net:  Your host is TAL.DE.EU.GameSurge.net, running version u2.10.12.18(gs2)
08.09. 21:28:52.639    DoGenericServerMessageReceived ** TAL.DE.EU.GameSurge.net:  This server was created Fri Nov 10 2017 at 11:10:57 CET
08.09. 21:28:52.640    DoGenericServerMessageReceived ** TAL.DE.EU.GameSurge.net:  There are 10 users and 1962 invisible on 13 servers
08.09. 21:28:52.640    DoGenericServerMessageReceived ** TAL.DE.EU.GameSurge.net: 27 operator(s) online
08.09. 21:28:52.640    DoGenericServerMessageReceived ** TAL.DE.EU.GameSurge.net: 1587 channels formed
08.09. 21:28:52.722    DoGenericServerMessageReceived ** TAL.DE.EU.GameSurge.net:  I have 1 clients and 1 servers
08.09. 21:28:52.723    DoGenericServerMessageReceived ** TAL.DE.EU.GameSurge.net Grant Highest connection count: 2 (1 clients) 
08.09. 21:28:52.723    DoGenericServerMessageReceived ** TAL.DE.EU.GameSurge.net Grant on 2 ca 1(10) ft 10(10) tr 
````